src/infrastructure/logger/index.js

```diff
@@ -26,14 +21,14 @@ const customLevels = {
     audit: 'magenta',
   },
 };
-
 const loggerConfig = {
   levels: customLevels.levels,
   transports: [],
 };
-
-// loggerConfig.transports.push(new winston.transports.Console({ level: logLevel, colorize: true }));
-
+const sequelizeTransport = WinstonSequelizeTransport(config);
```

Comment:
Is there any particular reason to replace  `AuditTransporter` with `WinstonSequelizeTransport`?

Reply:
Logging to database was not working, the package is missing as used in Profile and other components login.dfe.audit.winston-sequelize-transport

Reply:
Services component is logging correctly to Prod DB and looks like there is an infrastructure problem not publishing to the lower envs DBs, which needs to be investigated. 
`AuditTransporter` and and queues should be used everywhere and all the instances where `WinstonSequelizeTransport` is used should be replaced.